add missing Issues content for Review List GA (#3255)
getsentry/sentry-docs: src/docs/product/error-monitoring/index.mdx

---
title: Issues
sidebar_order: 1
description: "Learn more about the Issues page and how you can use it to efficiently triage issues."
---

> The Issues page of [sentry.io](https://sentry.io) is where you can view, [triage](/product/error-monitoring/issue-details/), and resolve error events sent by your application. A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.

Reviewer: I’d like us to make this an _extremely_ brief summary, like the Performance homepage description

Reviewer: Lots of repetitive use of "sentry.io" here

Reviewer: # Images

> The Issues homepage is where you can find important groups of errors in your app. Filter by event properties such as level, browser, device, the user that error impacted, unhandled errors, etc. You can then [triage them](link) or inspect the [issue details](link) to understand that problem in more depth.

<img width="1420" alt="CleanShot 2021-03-17 at 16 23 34@2x" src="https://user-images.githubusercontent.com/1900676/111551531-a5d6ba80-873d-11eb-8699-722ce321dd32.png">

### Issues are Groups of Error Events
A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, issues are grouped into similar events based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app; issues allow you to see how frequently that problem is happening and how many users it’s affecting. 

With this information, you can then make decisions about what to do next by [triaging the issue](link). For each issue, the page displays:

- Error type and message
- Brief description of the issue
- Associated project
- Issue timing, such as last and first time seen

### Organizing Issues 
The issues page is organized into tabs to help you group issues together:

- All Unresolved (is:unresolved): All unresolved issues, including issues in the "For Review" tab
- For Review (is:unresolved is:for_review): Issues that need to be reviewed
- Ignored (is:ignored): All ignored issues
- Saved Searches: Select from a set of premade and custom saved searches. The name of the tab changes based on your selection.

### Other Links
- Issue Details
- Using Breadcrumbs
- Issue States and Triage
- Issue Ownership Rules
- Reprocessing

> The Issues page of [sentry.io](https://sentry.io) is where you can view, [triage](/product/error-monitoring/issue-details/), and resolve error events sent by your application. A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.

Reviewer: "sentry.io" is repeated multiple times unnecessarily I reckon

> The Issues page of [sentry.io](https://sentry.io) is where you can view, [triage](/product/error-monitoring/issue-details/), and resolve error events sent by your application. A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.

Author: ```suggestion
The Issues page is where you can find information about important groups of errors in your application. Filter by error event properties such as browser, device, impacted users, or whether the error is unhandled. You can then inspect issue details to better understand the problem and [triage](/product/error-monitoring/issue-details/) effectively.

TO DO add screenshot here of issue stream
```

> 

Reviewer: We should have a picture of the Issues page directly beneath the first paragraph

The Issues page is organized into tabs, each corresponding to a filtered list of issues:

- All Unresolved (`is:unresolved`): All unresolved issues, including issues in the "For Review" tab
- For Review (`is:unresolved is:for_review`): Issues that need to be reviewed
- Ignored (`is:ignored`): All ignored issues
- Saved Searches: Select from a set of [premade](/product/sentry-basics/search/#premade-searches) and custom [saved searches](/product/sentry-basics/search/#organization-wide-saved-searches). The name of the tab changes based on your selection.

TO DO add screenshot here of issue stream

From this page, you can begin to triage an issue. Learn more about triaging issues and their different states in [Issue States and Triage](/product/error-monitoring/states-triage/).

For each issue, the page displays:

- Error type and message
- Brief description of the issue
- Associated project
- Issue timing, such as last and first time seen

> 

Reviewer: We should have a zoomed in picture of a single issue here perhaps

> The Issues page is organized into tabs, each corresponding to a filtered list of issues:
> 
> - All Unresolved (`is:unresolved`): All unresolved issues, including issues in the "For Review" tab
> - For Review (`is:unresolved is:for_review`): Issues that need to be reviewed
> - Ignored (`is:ignored`): All ignored issues
> - Saved Searches: Select from a set of [premade](/product/sentry-basics/search/#premade-searches) and custom [saved searches](/product/sentry-basics/search/#organization-wide-saved-searches). The name of the tab changes based on your selection.
> 
> TO DO add screenshot here of issue stream
> 
> From this page, you can begin to triage an issue. Learn more about triaging issues and their different states in [Issue States and Triage](/product/error-monitoring/states-triage/).
> 
> For each issue, the page displays:
> 
> - Error type and message
> - Brief description of the issue
> - Associated project
> - Issue timing, such as last and first time seen
> 

Author: ```suggestion

## Error Events and Issues

A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.

From the Issues page, you can begin to triage an issue. Learn more about triaging issues and their different states in [Issue States and Triage](/product/error-monitoring/states-triage/).

For each issue, the page displays:

- Error type and message
- Brief description of the issue
- Associated project
- Issue timing, such as last and first time seen

## Issues Page Tabs

The Issues page is organized into tabs, each corresponding to a filtered list of issues:

- All Unresolved (`is:unresolved`): All unresolved issues, including issues in the "For Review" tab
- For Review (`is:unresolved is:for_review`): Issues that need to be reviewed
- Ignored (`is:ignored`): All ignored issues
- Saved Searches: Select from a set of [premade](/product/sentry-basics/search/#premade-searches) and custom [saved searches](/product/sentry-basics/search/#organization-wide-saved-searches). The name of the tab changes based on your selection.

These different lists also help you with the triaging process. 

```

Learn more about investigating and routing issues:

> 

Reviewer: Might be worth having a breakdown of search on this page – what tags/properties you can search for

---
title: Issues
sidebar_order: 1

> description: "Learn more about how the configuration of your SDK helps you resolve issues quickly in [sentry.io](https://sentry.io)."

Reviewer 2: yaml frontmatter doesn't allow us to create the link to Sentry here.
```suggestion
description: "Learn more about the Issues page and how you can use it to efficiently triage issues."
```

---

> The Issues page of [sentry.io](https://sentry.io) is where you can view, triage, and resolve error events sent by your application. A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.

Reviewer 2: ```suggestion
The Issues page of [sentry.io](https://sentry.io) is where you can view, [triage](/product/error-monitoring/issue-details/), and resolve error events sent by your application. A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.
```

The Issues page is organized into tabs, each corresponding to a filtered list of issues:

> - All Unresolved (`is:unresolved`): All unresolved issues, including issues in the For Review tab

Reviewer 2: ```suggestion
- All Unresolved (`is:unresolved`): All unresolved issues, including issues in the "For Review" tab
```

- For Review (`is:unresolved is:for_review`): Issues that need to be reviewed
- Ignored (`is:ignored`): All ignored issues

> - Saved Searches: Select from a set of pre-built and custom saved searches. The name of the tab changes based on your selection.

Reviewer 2: ```suggestion
- Saved Searches: Select from a set of [premade](https://docs.sentry.io/product/sentry-basics/search/#premade-searches) and custom [saved searches](/product/sentry-basics/search/#organization-wide-saved-searches). The name of the tab changes based on your selection.
```

TO DO identify what link for last bullet above should be and add it
TO DO add screenshot here of issue stream

> From this page, you can begin to an issue. For information about triaging issues and their different states, see [Issue States and Triage](/product/error-monitoring/states-triage/).

Reviewer 2: ```suggestion
From this page, you can begin to triage an issue. Learn more about triaging issues and their different states in [Issue States and Triage](/product/error-monitoring/states-triage/).
```

> For each issue, the page displays several pieces of information including the following:

Reviewer 2: ```suggestion
For each issue, the page displays:
```

- Error type and message
- Brief description of the issue
- Associated project

> - Last time the issue was seen
> - First time the issue was seen

Reviewer 2: Perhaps this might be better captured as follows to allow for changes in the UI or expansion?
```suggestion
- Issue timing, such as last and first time seen
```

TO DO add screenshot here of issue stream

## Error Events and Issues

A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.

From the Issues page, you can begin to triage an issue. Learn more about triaging issues and their different states in [Issue States and Triage](/product/error-monitoring/states-triage/).

For each issue, the page displays:

- Error type and message
- Brief description of the issue
- Associated project
- Issue timing, such as last and first time seen

> ## Issues Page Tabs

Reviewer 3: This section can go away, let's discuss tabs in the triage page under the EA section
